crates/autopilot/src/database/auction.rs

```diff
@@ -0,0 +1,161 @@
+use super::Postgres;
+use anyhow::{anyhow, Context, Result};
+use database::{
+    auction::AuctionId,
+    orders::{
+        BuyTokenDestination as DbBuyTokenDestination, OrderKind as DbOrderKind,
+        SellTokenSource as DbSellTokenSource, SigningScheme as DbSigningScheme,
+    },
+    solver_competition::SolverCompetitionId,
+};
+use futures::{StreamExt, TryStreamExt};
+use model::{
+    app_id::AppId,
+    auction::Auction,
+    order::{
+        BuyTokenDestination, Order, OrderData, OrderKind, OrderMetadata, OrderStatus, OrderUid,
+        SellTokenSource,
+    },
+    signature::{Signature, SigningScheme},
+};
+use number_conversions::{big_decimal_to_big_uint, big_decimal_to_u256};
+use primitive_types::H160;
+
+pub struct SolvableOrders {
+    pub orders: Vec<Order>,
+    pub latest_settlement_block: u64,
+}
+
+impl Postgres {
+    pub async fn solvable_orders(&self, min_valid_to: u32) -> Result<SolvableOrders> {
```

Comment:
This function is copied from the orderbook crate where it can be removed when the autopilot takes over.